src/euroeval/benchmarker.py

```diff
@@ -413,6 +419,12 @@ def benchmark(
                             if benchmark_config.raise_errors:
                                 raise e
                             logger.info(e.message)
+                            remaining_tasks = (
+                                len(dataset_configs)
+                                - dataset_configs.index(dataset_config)
+                                - 1
+                            )
+                            current_benchmarks += remaining_tasks
```

Comment:
A task can consist of multiple datasets, so this is misleading:

```suggestion
                            # Add the remaining number of benchmarks for the model to our
                            # benchmark counter, since we're skipping the rest of them
                            current_benchmarks += (
                                len(dataset_configs)
                                - dataset_configs.index(dataset_config)
                                - 1
                            )
```